File: lesson4/c_c++/homework/homework4_3.cpp
#include <iostream>

int main() {
    using namespace std;
    int numb, result = 0;

    cout << "Введите натуральное число => ";
    cin >> numb;

    int numb_u = 1;
    while (numb_u <= numb) {
        result += numb_u;
        numb_u++;
    }
    cout << "Сумма натуральных чисел => " << result << '\n';
    return 0;
}

File: lesson4/python3/homework/homework4_3.py
#!/usr/bin/python3

numb = int(input("Введите натуральное число => "))
numb_u = 0

numb_i = 1
while numb_i <= numb:
    numb_u += numb_i
    numb_i += 1
print("Сумма натурального числа =>", numb_u)

